### PR TITLE
common: remove Ubuntu 16.04 and unneeded scripts for installing ndctl and pmdk

### PR DESCRIPTION
**Pull request by ldorau**
This change is [Reviewable](https://reviewable.io/reviews/pmem/rpma/871)

### CODE REVIEW

**Review by janekmi — CHANGES_REQUESTED** [review]
> Reviewed 7 of 7 files at r1.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: all files reviewed, 1 unresolved discussion (waiting on @ldorau)

*[a discussion](https://reviewable.io/reviews/pmem/rpma/871#-MTAVEbi1EM0COe3vzX8:-MTAVEbi1EM0COe3vzX9:bvpcwzw) (no related file):*
https://github.com/pmem/rpma/blame/master/INSTALL.md#L136

---

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: 7 of 8 files reviewed, 1 unresolved discussion (waiting on @janekmi)

*[a discussion](https://reviewable.io/reviews/pmem/rpma/871#-MTAVEbi1EM0COe3vzX8:-MTAWgRY7RZNS0YYqjtS:b-896fix) (no related file):*
<details><summary><i>Previously, janekmi (Jan Michalski) wrote…</i></summary><blockquote>

https://github.com/pmem/rpma/blame/master/INSTALL.md#L136
</blockquote></details>

Done.

---

**Review by janekmi — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871#-:-MTAY_M6-zH09_CenSN3:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ldorau)

**Review by grom72 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: all files reviewed, 1 unresolved discussion (waiting on @ldorau)

___
*[INSTALL.md, line 143 at r2](https://reviewable.io/reviews/pmem/rpma/871#-MTAkdh94INc2WaGStl3:-MTAkdh94INc2WaGStl4:bvwb0th) ([raw file](https://github.com/pmem/rpma/blob/f3518433c5d23491d7833d0a9d173f8783611574/INSTALL.md#L143)):*
> ```
> OSes not tested anymore:
> ```
Known supported but not validated OSes:

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: all files reviewed, 1 unresolved discussion (waiting on @grom72)

___
*[INSTALL.md, line 143 at r2](https://reviewable.io/reviews/pmem/rpma/871#-MTAkdh94INc2WaGStl3:-MTAxCPl6kRV6EMgLDST:b77ab46) ([raw file](https://github.com/pmem/rpma/blob/f3518433c5d23491d7833d0a9d173f8783611574/INSTALL.md#L143)):*
<details><summary><i>Previously, grom72 (Tomasz Gromadzki) wrote…</i></summary><blockquote>

> ```
> OSes not tested anymore:
> ```
Known supported but not validated OSes:
</blockquote></details>

No, we are not sure they will be supported. They can become unsupported and we will not get to know that, because we are not testing them anymore.

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: all files reviewed, 1 unresolved discussion (waiting on @grom72)

___
*[INSTALL.md, line 143 at r2](https://reviewable.io/reviews/pmem/rpma/871#-MTAkdh94INc2WaGStl3:-MTAyDGb8Hhk8BQhe6MF:b-32xm39) ([raw file](https://github.com/pmem/rpma/blob/f3518433c5d23491d7833d0a9d173f8783611574/INSTALL.md#L143)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

No, we are not sure they will be supported. They can become unsupported and we will not get to know that, because we are not testing them anymore.
</blockquote></details>

So rather something like:

"OSes that were supported, but we stopped to validate them:"

**Review by janekmi — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: all files reviewed, 1 unresolved discussion (waiting on @grom72)

___
*[INSTALL.md, line 143 at r2](https://reviewable.io/reviews/pmem/rpma/871#-MTAkdh94INc2WaGStl3:-MTB43EB3gFJfy_NqRfX:b4rvpdq) ([raw file](https://github.com/pmem/rpma/blob/f3518433c5d23491d7833d0a9d173f8783611574/INSTALL.md#L143)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

So rather something like:

"OSes that were supported, but we stopped to validate them:"
</blockquote></details>

In short: `OSes not tested anymore`. ;-)

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: all files reviewed, 1 unresolved discussion (waiting on @grom72)

___
*[INSTALL.md, line 143 at r2](https://reviewable.io/reviews/pmem/rpma/871#-MTAkdh94INc2WaGStl3:-MTB60HfAwwOajPPMBf-:b2nh4j5) ([raw file](https://github.com/pmem/rpma/blob/f3518433c5d23491d7833d0a9d173f8783611574/INSTALL.md#L143)):*
<details><summary><i>Previously, janekmi (Jan Michalski) wrote…</i></summary><blockquote>

In short: `OSes not tested anymore`. ;-)
</blockquote></details>

:-)

**Review by janekmi — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: all files reviewed, 1 unresolved discussion (waiting on @grom72)

**Review by grom72 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: all files reviewed, 1 unresolved discussion (waiting on @janekmi)

___
*[INSTALL.md, line 143 at r2](https://reviewable.io/reviews/pmem/rpma/871#-MTAkdh94INc2WaGStl3:-MTEY2CyBWjeIGt7pzjE:b-p9apzs) ([raw file](https://github.com/pmem/rpma/blob/f3518433c5d23491d7833d0a9d173f8783611574/INSTALL.md#L143)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

:-)
</blockquote></details>

So they are not supported anymore and no point to list them here. 
I would propose to add the removal of Ubuntu 16.04 to https://github.com/pmem/rpma/blob/master/CHANGELOG.md.

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: 7 of 8 files reviewed, 1 unresolved discussion (waiting on @grom72 and @janekmi)

___
*[INSTALL.md, line 143 at r2](https://reviewable.io/reviews/pmem/rpma/871#-MTAkdh94INc2WaGStl3:-MTEsSVbEUcOe-EtclUY:bps9ovg) ([raw file](https://github.com/pmem/rpma/blob/f3518433c5d23491d7833d0a9d173f8783611574/INSTALL.md#L143)):*
<details><summary><i>Previously, grom72 (Tomasz Gromadzki) wrote…</i></summary><blockquote>

So they are not supported anymore and no point to list them here. 
I would propose to add the removal of Ubuntu 16.04 to https://github.com/pmem/rpma/blob/master/CHANGELOG.md.
</blockquote></details>

Removed.

The changelog should be updated before the release - it is not a part of this PR.

**Review by grom72 — COMMENTED** [review]
:lgtm:

> Reviewed 6 of 7 files at r1, 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871#-:-MTFGuGg8GZoPIi5T7XP:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ldorau)

**Review by grom72 — COMMENTED** [review]
> Reviewed 1 of 7 files at r1.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/871)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ldorau)
